include/oneapi/dpl/internal/serial_numeric_impl.h

```diff
@@ -22,8 +22,7 @@ namespace oneapi
 {
 namespace dpl
 {
-//namespace serial {
-
+#if __cplusplus > 201402L
```

Comment:
I believe there should be a conflict with standard implementation since `reduce` is in standard since C++17